manifests/testing/rbac-for-testing.yaml.in

```diff
@@ -22,3 +22,42 @@ subjects:
   - kind: ServiceAccount
     name: kubevirt-testing
     namespace: {{ namespace }}
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: kubevirt-subresource-test-user
+  namespace: default
```

Comment:
change to {{ namespace }}

Reply:
i was able to remove this service account. It was used by the functional tests. I added code to dynamically generate the service account similar to how the iscsi pvcs are created/destroyed every test run.